odoo/addons/base/tests/common.py

```diff
@@ -87,7 +87,7 @@ def setup_independent_user(cls):
 
     @classmethod
     def get_default_groups(cls):
-        return cls.env['res.users']._default_groups()
+        return cls.env['res.groups']
```

Comment:
Is it really an issue to keep the default groups here ? :thinking: 
The test infrastructure should reflect the default behavior IMO.

If you really want to use it to enforce the default user to not have the demo groups, at least keep the `base.group_user` here.

Reply:
As you said it's only really used in accounting, so it shouldn't matter much, I return an empty recordset such that overriding the method is easy, all solutions are valid imo.

Reply:
Yes, but the mid/long term objective is to remove all the custom groups in accounting and ideally rely on this default behavior, so I believe we should keep the default groups here.

Reply:
Sounds good.